content/en/logs/explorer/group.md

```diff
@@ -109,6 +109,21 @@ Patterns support the [List Aggregates][15] visualization. Clicking a pattern in
 
 {{< img src="logs/explorer/patterns_side_panel.jpg" alt="The log side panel with the view all button and the parsing rule highlighted" style="width:80%;" >}}
 
+### Pattern Inspector
+
+Use Pattern Inspector to get a visual breakdown of the underlying values of a log pattern’s aggregation based on your search query. For example, if you are investigating an issue, you could see how many hosts are involved or what regions or data centers are impacted.
+
+{{< img src="logs/explorer/group/inspect_values.png" alt="The distribution of values pop-up showing a bar graph of the values" style="width:70%;" >}}
```

Comment:
In my experience, most style guides recommend not using pop-up as a noun. For example, you can see this reflected in the Microsoft Style Guide and Rackspace Style Guides (both available for free online). That said, it doesn't appear that we have a style guideline for this particular term. And I'm also struggling to come up with a better solution, other than "pop-up window".

Reply:
Thanks @drichards-87! Ah yeah, we usually don't use pop-up either, I forgot 🙃. I changed it to `graph`. Also applied your other suggestions. Could you re-review?